tests/unit/views/test_model_view.py

```diff
@@ -41,41 +41,33 @@ class ItemView(RetrieveUpdateDestroyModelView):
 
 
 @pytest.fixture
-def database():
-    db = SqliteDatabase(':memory:')
-    db.bind(MODELS)
-    db.connect()
-    db.create_tables(MODELS)
+def database(database_maker):
+    return database_maker(models=MODELS)
 
 
 @pytest.fixture
-def http(loop, aiohttp_client):
-    middlewares = [
-        handle_errors_json
-    ]
-
-    app = web.Application(middlewares=middlewares)
-    app.router.add_route('*', '/paginated-items', ItemsPaginatedView)
-    app.router.add_route('*', '/items', ItemsView)
-    app.router.add_route('*', '/items/{id}', ItemView)
-    return loop.run_until_complete(aiohttp_client(app))
+async def model_view_http_client(http_client_maker):
```

Comment:
Can we have a more simple name for this fixture? `model_view` (from the name) looks a bit redundant (we're already in a module called `test_model_view`). What about `http_client` or simple `http`?

Reply:
Hmm at first I thought `pytest` fixtures are global and name clashes are to be avoided; that's why I prefixed them like this. Now I'm not sure about it. What do you know about this? Do you know for a fact that fixtures are scoped to the module or package where they are defined?

Reply:
As far as I know, if a fixture is defined in the test module, that's a "module level fixture" so we shouldn't have conflicts. We should make sure that that's how it works, I'm not 100% sure.